packages/apollo-server-errors/src/index.ts

```diff
@@ -38,6 +38,7 @@ export class ApolloError extends Error implements GraphQLError {
 
 function enrichError(error: Partial<GraphQLError>, debug: boolean = false) {
   const expanded = {} as any;
+  Object.setPrototypeOf(expanded, Object.getPrototypeOf(error));
```

Comment:
Or use `Object.create(Object.getPrototypeOf(error), properties)`, with `properties` being the same object you pass to `Object.defineProperties` (so you don't need a separate call for that anymore).